index4.js

```diff
@@ -0,0 +1,12 @@
+// Напишите функцию isEven(n), которая принимает один параметр, число, и возвращает, четное ли оно. Т.е. функция может 
+// возвращать два значения: четное или нечетное.
+
+const isEven = function (number) {
+    if (number % 2 === 0) {
+        return true;;
+    }
+    
+    return false
```

Comment:
for this task is enough to pass return number % 2 === 0, because this comparison return tru or false anyway